diff --git i/src/mcp_server_fetch/server.py w/src/fetch/src/mcp_server_fetch/server.py
index 2df9d3b..d128987 100644
--- i/src/mcp_server_fetch/server.py
+++ w/src/mcp_server_fetch/server.py
@@ -70,11 +70,11 @@ async def check_may_autonomously_fetch_url(url: str, user_agent: str, proxy_url:
     """
     from httpx import AsyncClient, HTTPError

     robot_txt_url = get_robots_txt_url(url)

-    async with AsyncClient(proxies=proxy_url) as client:
+    async with AsyncClient(proxy=proxy_url) as client:
         try:
             response = await client.get(
                 robot_txt_url,
                 follow_redirects=True,
                 headers={"User-Agent": user_agent},
@@ -114,11 +114,11 @@ async def fetch_url(
     """
     Fetch the URL and return the content in a form ready for the LLM, as well as a prefix string with status information.
     """
     from httpx import AsyncClient, HTTPError

-    async with AsyncClient(proxies=proxy_url) as client:
+    async with AsyncClient(proxy=proxy_url) as client:
         try:
             response = await client.get(
                 url,
                 follow_redirects=True,
                 headers={"User-Agent": user_agent},
