diff --git c/src/solidlsp/language_servers/rust_analyzer.py i/src/solidlsp/language_servers/rust_analyzer.py
index 48bae28..ad9fb7d 100644
--- c/src/solidlsp/language_servers/rust_analyzer.py
+++ i/src/solidlsp/language_servers/rust_analyzer.py
@@ -29,11 +29,11 @@ class RustAnalyzer(SolidLanguageServer):
         self, config: LanguageServerConfig, logger: LanguageServerLogger, repository_root_path: str, solidlsp_settings: SolidLSPSettings
     ):
         """
         Creates a RustAnalyzer instance. This class is not meant to be instantiated directly. Use LanguageServer.create() instead.
         """
-        rustanalyzer_executable_path = self._setup_runtime_dependencies(logger, config, solidlsp_settings)
+        rustanalyzer_executable_path = "rust-analyzer"
         super().__init__(
             config,
             logger,
             repository_root_path,
             ProcessLaunchInfo(cmd=rustanalyzer_executable_path, cwd=repository_root_path),
